vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -48,27 +48,32 @@ You can use `ent help` to review the list of available tools. Check the help tex
  Entando CLI tools
 ~~~~~~~~~~~~~~~~~~~
 
+> Essentials:
```

Comment:
IMO this should be changed to the generic home syntax. For example,

`Activate using:  ~/.entando/ent/quickstart/cli/v6.3.0/activate`

Reply:
Yeah, it's the actual output from the command so I was resisting editing it. You'd actually get this from inside the VM.

Reply:
Fixed